cpu_check.cc

```diff
@@ -714,8 +715,8 @@ void Worker::Run() {
 
     LOG_EVERY_N_SECS(INFO, 30) << Jstat(
         Json("elapsed_s", static_cast<uint64_t>(TimeInSeconds() - t0)) + ", " +
-        Json("failures", errorCount.load()) + ", " +
-        Json("successes", successCount.load()) + ", " + Writer() +
+        Json("failures", static_cast<double>(errorCount.load())) + ", " +
```

Comment:
Why double for these two?

Reply:
I thought load instance returns a double type

Reply:
i updated see below